spec/index.bs

```diff
@@ -412,6 +412,34 @@ for the [=usage scope=] of this render pass.
 
 Issue(gpuweb/gpuweb#514): Document read-only states for depth views.
 
+## Synchronization ## {#programming-model-synchronization}
+
+For each [=subresource=] of a [=physical resource=], the usage is tracked on the [=Queue timeline=].
+
+Timeline could be split into a sequence of [=usage scope=]:
+within each scope the usage of the [=subresource=] stays unchanged,
```

Comment:
```suggestion
The [=Queue timeline=] is split into a sequence of [=usage scope=]s.
Within each scope the usage of the [=subresource=] stays unchanged,
```

Reply:
In multiqueue, there's one queue timeline per queue, right? Hard to say now, but there needs to be at least some synchronization at the device timeline.

Reply:
I think it would be up to the multi-queue proposal to specify.